clouds/snowflake/modules/doc/h3/H3_POLYFILL.md

```diff
@@ -1,15 +1,21 @@
 ## H3_POLYFILL
 
 ```sql:signature
-H3_POLYFILL(geography, resolution)
+H3_POLYFILL(geography, resolution)  
```

Comment:
we normally use a different notation for telling that some parameters are optional:

```sql:signature
H3_POLYFILL(geography, resolution [, mode])
```